tsconfig.json

```diff
@@ -2,10 +2,16 @@
   "compilerOptions": {
     "sourceMap": true,
     "target": "esnext",
+    "module": "commonjs",
+    "outDir": "build",
+    "declaration": true,
+    "esModuleInterop": true,
     "removeComments": false,
-    "allowSyntheticDefaultImports": true
+    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
```

Comment:
The reason for adding `skipLibCheck` is that the following issues occur during the build.
```bash
> yorkie-js-sdk@0.0.10 build:publish /Users/choedongcheol/Workspace/opensource/yorkie-js-sdk
> tsc

src/api/yorkie_pb.d.ts:1041:21 - error TS2315: Type 'Array' is not generic.

1041     getNodesList(): Array<RHTNode>;
                         ~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1042:25 - error TS2315: Type 'Array' is not generic.

1042     setNodesList(value: Array<RHTNode>): Object;
                             ~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1071:18 - error TS2315: Type 'Array' is not generic.

1071       nodesList: Array<RHTNode.AsObject>,
                      ~~~~~~~~~~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1080:21 - error TS2315: Type 'Array' is not generic.

1080     getNodesList(): Array<RGANode>;
                         ~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1081:25 - error TS2315: Type 'Array' is not generic.

1081     setNodesList(value: Array<RGANode>): Array;
                             ~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1110:18 - error TS2315: Type 'Array' is not generic.

1110       nodesList: Array<RGANode.AsObject>,
                      ~~~~~~~~~~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1162:21 - error TS2315: Type 'Array' is not generic.

1162     getNodesList(): Array<TextNode>;
                         ~~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1163:25 - error TS2315: Type 'Array' is not generic.

1163     setNodesList(value: Array<TextNode>): Text;
                             ~~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1192:18 - error TS2315: Type 'Array' is not generic.

1192       nodesList: Array<TextNode.AsObject>,
                      ~~~~~~~~~~~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1201:21 - error TS2315: Type 'Array' is not generic.

1201     getNodesList(): Array<RichTextNode>;
                         ~~~~~~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1202:25 - error TS2315: Type 'Array' is not generic.

1202     setNodesList(value: Array<RichTextNode>): RichText;
                             ~~~~~~~~~~~~~~~~~~~

src/api/yorkie_pb.d.ts:1231:18 - error TS2315: Type 'Array' is not generic.

1231       nodesList: Array<RichTextNode.AsObject>,
```

I had the same problem before. So when declaring the typescript Array type, it was used in the same way as `any[]`.

I am not currently familiar with this issue. So I am looking for the cause.